project/ws/app/src/lib/routes/home/routes/users-view/single-user-creation/single-user-creation.component.ts

```diff
@@ -216,6 +216,12 @@ export class SingleUserCreationComponent implements OnInit, AfterViewInit, OnDes
     }
   }
 
+  handleValidTags(event: any): any {
+    const charCode = event.charCode
```

Comment:
are we showing error message, if tag is invalid?

Reply:
Nope! We are not allowing them to type the special character only. So no point of showing error message at all

Reply:
please check with product